lib/rules/no-irregular-whitespace.js

```diff
@@ -18,13 +19,20 @@ module.exports = function(context) {
     // Module store of errors that we have found
     var errors = [];
 
+    // Comment nodes.  We accumulate these as we go, so we can be sure to trigger them after the whole `Program` entity is parsed, even for top-of-file comments.
+    var commentNodes = [];
+
+    // Lookup the `skipComments` option, which defaults to `false`.
+    var options = context.options[0] || {};
+    var skipComments = typeof options.skipComments === "undefined" ? false : !!options.skipComments;
```

Comment:
Dang.  Right, it can be simplified now.  Just pushed an updated PR.

--
Christophe Porteneuve
tdd@tddsworld.com